code/model/modelinterp.cpp

```diff
@@ -1273,12 +1256,10 @@ void submodel_get_two_random_points_better(int model_num, int submodel_num, vec3
 			submodel_num = pm->detail[0];
 		}
 
-		// the Shivan Comm Node does not have a collision tree, for one
 		if (pm->submodel[submodel_num].collision_tree_index < 0) {
 			nprintf(("Model", "In submodel_get_two_random_points_better(), model %s does not have a collision tree!  Falling back to submodel_get_two_random_points().\n", pm->filename));
```

Comment:
a) I have never, ever seen this print
b) the function name referenced here should be updated to reflect the actual function throwing this warning